cardano-testnet/test/cardano-testnet-test/Cardano/Testnet/Test/Gov/DRepActivity.hs

```diff
@@ -7,6 +7,7 @@
 
 module Cardano.Testnet.Test.Gov.DRepActivity
   ( hprop_check_drep_activity
+  , makeActivityChangeProposal
```

Comment:
We shouldn't make cross-dependencies between test modules IMO. It would be better to move this function to a test source module.

Reply:
Addressed as part of: https://github.com/IntersectMBO/cardano-node/pull/5871#discussion_r1630106090